src/OrchardCore.Modules/OrchardCore.Deployment/Models/FileDownloadDeploymentTargetSettings.cs

```diff
@@ -0,0 +1,15 @@
+namespace OrchardCore.Deployment.Models
+{
+    public class FileDownloadDeploymentTargetSettings
+    {
+        /// <summary>
+        /// The name of the rsa secret that can be used to encrypt exported data.
+        /// </summary>
+        public string RsaEncryptionSecret { get; set; }
+
+        /// <summary>
+        /// The name of the rsa secret that can be used to sign encrypted exported data.
```

Comment:
```suggestion
        /// The name of the RSA secret that can be used to encrypt exported data.
        /// </summary>
        public string RsaEncryptionSecret { get; set; }

        /// <summary>
        /// The name of the RSA secret that can be used to sign encrypted exported data.
```

Reply:
Okay I agree, I did it in some other places.